### ISSUE TITLE
Visual glitch

### ISSUE THREAD

**Issue by bazsi0716** [MyM-ModpackTeam/GalacticScience#203]
I downloaded the modpack on my notebook, and when I open a new world everything is blurred, but when I scroll to an empty space on my inventory everything is ok. But some chunk is glitched all the time.

**Comment by Aiidoneus** [comment]
Can you screenshot what you are seeing? I can't reproduce this

**Comment by JoaoPedro159** [comment]
> i think this is it.
I have a notebook too and when I created and join in a new world happened this screen, I still can hear the sounds of mining and running but I can't see anything(image1).
Wait... I dropped a item and I can see, but when I'm holding any item all my graphics glitches(image2), I think it's a visual bug or something like this.
If you need any more information ou details just tell me and I see what I can do.
![image](https://user-images.githubusercontent.com/75337624/190933915-85653973-36b4-464f-a2f1-a39989cdeaf5.png)
![image](https://user-images.githubusercontent.com/75337624/190934006-59fdbfc6-4b94-46a0-a66e-f13dbdc8e612.png)

**Comment by bazsi0716** [comment]
Yes exactly, this is my problem too. When I hold an item, I can't see anything because of the 'blure', but when I drop the item from my hotbar, there are a lots of visual glitch.

**Comment by JoaoPedro159** [comment]
> Yes exactly, this is my problem too. When I hold an item, I can't see anything because of the 'blure', but when I drop the item from my hotbar, there are a lots of visual glitch.

And you find a way to fix that?

**Comment by KEKE-vas** [comment]
I have the exact same problem and I can't find anything anywhere about it

**Comment by ImFlopp** [comment]
I'm assuming you are uing intel drivers.
Downgrade your intel graphics driver to something older than early this year. I am now using version 30.0.100.9955 from october 2021 and that actually fixed it.

Make sure this is the correct driver for you, but I got it from here: https://www.intel.com/content/www/us/en/download/19344/677976/intel-graphics-windows-dch-drivers.html

In the top, you can choose the version. Running the exe takes a while but then the intel installer pops up and checking in your device manager, you should see the current driver version. Hope that helps everyone else.

**Comment by sanato96** [comment]
> I'm assuming you are uing intel drivers. Downgrade your intel graphics driver to something older than early this year. I am now using version 30.0.100.9955 from october 2021 and that actually fixed it.
> 
> Make sure this is the correct driver for you, but I got it from here: https://www.intel.com/content/www/us/en/download/19344/677976/intel-graphics-windows-dch-drivers.html
> 
> In the top, you can choose the version. Running the exe takes a while but then the intel installer pops up and checking in your device manager, you should see the current driver version. Hope that helps everyone else.

and if I use nvidia drivers? This did not happen on the same laptop in the years 20-21

**Comment by ImFlopp** [comment]
> and if I use nvidia drivers? This did not happen on the same laptop in the years 20-21

That shouldn't happen. I upgraded my pc to rtx 3060 some time ago and the game's alright. Try to upgrade or downgrade the drivers
